connect/mirror/src/main/java/org/apache/kafka/connect/mirror/MirrorCheckpointConfig.java

```diff
@@ -81,7 +80,7 @@ public class MirrorCheckpointConfig extends MirrorConnectorConfig {
 
     public MirrorCheckpointConfig(Map<String, String> props) {
         super(CONNECTOR_CONFIG_DEF, ConfigUtils.translateDeprecatedConfigs(props, new String[][]{
```

Comment:
we don't need to call `translateDeprecatedConfigs` now, since the deprecated configs are removed